crates/re_renderer/src/transform.rs

```diff
@@ -0,0 +1,79 @@
+//! Transformation utilities.
+//!
+//! Some space definitions to keep in mind:
+//!
+//! Texture coordinates:
+//! * origin top left
+//! * full texture ([left; right], [top; bottom]):
+//!     ([0; 1], [0; 1])
+//!
+//! NDC:
+//! * origin center
+//! * full screen ([left; right], [top; bottom]):
+//!     ([-1; 1], [1; -1])
+//!
+//! Pixel coordinates:
+//! * origin top left
+//! * full screen ([left; right], [top; bottom]):
+//!      ([0; `screen_extent.x`], [0; `screen_extent.y`])
```

Comment:
The fact that some coordinate systems has Y=up and other Y=down makes me question the wisdom of naming the member of `RectF32` `left_top` instead `min`